.github/workflows/verify-dashboards-alerts.yaml

```diff
@@ -0,0 +1,135 @@
+name: Verify Dashboards and Alerts OK
+
+on: 
+  push:
+    branches: main
+    paths:
+       # Dashboards
+       - examples/dashboards/app_developer.json
+       - examples/dashboards/business_user.json
+       - examples/dashboards/platform_engineer.json
+       # Alerts
+       - examples/alerts/prometheusrules_policies_missing.yaml
+       - examples/alerts/slo-availability.yaml
+       - examples/alerts/slo-latency.yaml
+jobs:
+  verify-dashboards-alerts:
+    name: Verify Dashboards and Alerts OK
+    runs-on: ubuntu-latest
+    defaults: 
+      run:
+        shell: bash -eo pipefail {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Quickstart
+        run: |
+          sudo chown $(whoami) .
+
+          sudo apt-get update -y
+          sudo apt-get install -y golang expect
+
+          export ISTIO_INSTALL_SAIL=true
+
+          cat << 'EOF' > automate_setup.expect
+          #!/usr/bin/expect -f
+          set timeout -1
+          spawn bash hack/quickstart-setup.sh
+          expect "Are you ready to begin? (y/n)"
+          send "y\r"
+          expect "Do you want to set up a DNS provider for use with Kuadrant's DNSPolicy API? (y/n)"
+          send "n\r"
+          expect "Thank you for using Kuadrant! If you have any questions or feedback, please reach out to our community."
+          EOF
+          chmod +x automate_setup.expect
+          ./automate_setup.expect
+
+      - name: Port forward grafana and check if contains dashboards.
+        run: |
+          # Port forward Grafana
+          kubectl -n monitoring wait --for=condition=available deployment grafana --timeout=600s
+          kubectl -n monitoring port-forward service/grafana 3000:3000 &
+          grafana_process_id=$!
+          echo "Successfully port forwarded Grafana service."
+
+          sleep 1
```

Comment:
Why? If it is really needed, are we sure sleep 1 is always enough? Isnt there a better way to check?

Reply:
https://github.com/Kuadrant/kuadrant-operator/pull/708#discussion_r1668332829